### DISCUSSION TITLE
K-means output results table

### DISCUSSION THREAD

**Discussion by darivast** [semiautomaticgit/SemiAutomaticClassificationPlugin#215] [Q&A]
Dear SCP team,

I'm wondering to know what is the last column of the output from the K-means method. From my understanding, there is a value for each band and for each class but there is another column at the end. Please need help to understand this. When I run ISODATA this column does not appear.

thanks in advance,

David

**Comment by semiautomaticgit** [comment]
Hello @darivast , the last column is the spectral distance calculated in the last iteration of the K-means clustering. 
The iteration is terminated if the distance is lower than a selected distance threshold.

**Reply by darivast** [reply]
Many thanks Luca! But why this column do not appear when using ISODATA?

**Comment by semiautomaticgit** [comment]
Yes, it could should be added also to the ISODATA output
